modules/sso.php

```diff
@@ -970,9 +970,19 @@ static function get_user_by_wpcom_id( $wpcom_user_id ) {
 	 * @return string
 	 **/
 	public function error_msg_enable_two_step( $message ) {
-		$err = __( sprintf( 'This site requires two step authentication be enabled for your user account on WordPress.com. Please visit the <a href="%1$s" target="_blank"> Security Settings</a> page to enable two step', 'https://wordpress.com/me/security/two-step' ) , 'jetpack' );
+		$error = sprintf(
+			wp_kses(
```

Comment:
It's needed, since the translatable string has HTML and its final output can't be sanitized with `esc_html` and someone could inject malicious HTML in the translated string. With `wp_cases`, the final output is safe.